pkgs/development/tools/google-app-engine/python-sdk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs }:
```

Comment:
This is poor style, make dependencies explicit by passing them as individual parameters.